### PR TITLE
ADD: About us page

### PR DESCRIPTION
**Pull request by nandini06-helix**

### CODE REVIEW

File: src/Pages/AboutUs/About.jsx [hunk 1]
```diff
@@ -1,10 +1,70 @@
...
+import { useState } from "react";
+import styles from "./About.module.scss";
+import About1 from "../../Assets/About/About1.svg";
+import About2 from "../../Assets/About/About2.svg";
+import dropArrow from "../../Assets/About/dropArrow.svg";
 const About = () => {
+  const [select, setSelect] = useState(null);
+  const data = [
+    {
+      question: "Can I check the status of my complaint in real time?",
```
Comment (hereisadi) [inline src/Pages/AboutUs/About.jsx:10]: add id as an unique key for the each FAQ entry

File: src/Pages/AboutUs/About.jsx [hunk 2]
```diff
@@ -1,10 +1,70 @@
...
+          to voice your grievances effortlessly. Not only that, but we also hold
+          authorities accountable with our transparent, step-by-step tracking system.
+          Vyatha is the future of hostel living, and we&apos;re just getting started. Join
+          us, and let&apos;s revolutionize your hostel experience together!
+        </p>
+      </div>
+      <img src={About2} className={styles.img2} alt="" />
+      <div className={styles.accordion}>
+        {data.map((item, i) => (
+          <div className={select === i ? styles.accordion_show : styles.accordion_hide}>
```
Comment (hereisadi) [inline src/Pages/AboutUs/About.jsx:55]: pass`key={item.id}`

File: src/Pages/AboutUs/About.jsx
Comment (hereisadi) [inline src/Pages/AboutUs/About.jsx:1]: you might also want to add some transition, when accordion expands

File: src/Components/Shared/CaptchaComponent/Captcha.jsx [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    const num2 = Math.floor(Math.random() * 10) + 1;
+    const answer = num1 + num2;
+    setNum1(num1);
+    setNum2(num2);
+    setAnswer(answer);
+    setUserResponse("");
+    // setMessage('');
+  }
+
+  function verifyCaptcha() {
```
Comment (hereisadi) [inline src/Components/Shared/CaptchaComponent/Captcha.jsx:28]: prevent the default behaviour of button onClick (i.e refresh)
also for generateCaptcha

**Review by hereisadi — CHANGES_REQUESTED** [review]

**Review by Himangshu-101 — CHANGES_REQUESTED** [review]
Write proper codes with CSS. If you're implementing mobile view design, use mobile responsive view tool properly.

**Comment by Himangshu-101** [comment]
image
This is an image from mobile view. The unusual position of 'Captcha div' is because you have either used 'vh' unit or fixed position of the div somewhere. Utilize the developer tool mobile view properly, and scale the views repeatedly to observe breakpoints to write mobile view code.

**Pull request merged**